src/components/App/app.scss

```diff
@@ -0,0 +1,19 @@
+@import url('https://fonts.googleapis.com/css?family=Raleway:100,100i,200,200i,300,300i,400,400i,500,500i,600,600i,700,700i,800,800i,900,900i&display=swap');
+
+*{
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-size: 16px;
+  font-family: 'Raleway', sans-serif;
```

Comment:
Wouldn’t it be better to declare the font properties as variables?

Reply:
Yes, it will be done in the future PR 👐